org-journal.el

```diff
@@ -273,7 +278,10 @@ Otherwise, date ascending."
 If no TIME is given, uses the current time."
   (file-truename
    (expand-file-name
-    (format-time-string org-journal-file-format time)
+    (format-time-string
+     (concat org-journal-file-format
```

Comment:
You could write the `(or (and org-journal-encrypt-journals ".gpg") "")` more succinctly as `(if org-journal-encrypt-journals ".gpg" "")`. (Or am I missing something?)

Reply:
Whoops, right! :-D